### PR TITLE
Support kotlin 1.9.20-Beta2

### PR DESCRIPTION
**Pull request by eymar**

### CODE REVIEW

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt [hunk 1]
```diff
@@ -16,7 +16,8 @@ internal object ComposeCompilerCompatibility {
         "1.9.0-RC" to "1.4.8-beta",
         "1.9.0" to "1.5.1",
         "1.9.10" to "1.5.2",
-        "1.9.20-Beta" to "1.5.2.1-Beta2"
+        "1.9.20-Beta" to "1.5.2.1-Beta2",
+        "1.9.20-Beta2" to "1.5.2.1-Beta3"
```
Comment (igordmn) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt:20]: In the future, we should name compiler versions the same as we name main compose versions (-beta01, -beta02)

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt [hunk 2]
```diff
@@ -16,7 +16,8 @@ internal object ComposeCompilerCompatibility {
...
     )
 
     fun compilerVersionFor(kotlinVersion: String): String {
```
Comment (igordmn) [inline gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt:23]: We have a failed test. Does it fail before the change?

**Review by igordmn — APPROVED** [review]

File: gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt (hunk 2 above)
Comment (eymar) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt:23]: I'll check.
This change doesn't affect the version of the plugin used in the tests. So it's likely a flaky test/environemnt.

Comment (eymar) [reply gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt:23]: rerun helped :|

**Pull request merged**